helm/grafana/Chart.yaml

```
@@ -12,6 +12,10 @@ dependencies:
   - name: grafana
     version: 9.2.1
     repository: https://grafana.github.io/helm-charts
+  - name: pg-cluster-recovery-test
```

Adding a condition makes this dependency optional, and only used when you actually need/want to use it.
```suggestion
  - name: pg-cluster-recovery-test
    condition: pgClusterRecoveryTest.enabled
```